packages/js/product-editor/src/components/details-categories-field/test/use-category-search.test.ts

```diff
@@ -215,15 +218,18 @@ describe( 'useCategorySearch', () => {
 			} );
 			await waitForNextUpdate();
 			expect( result.current.categoriesSelectList.length ).toEqual( 6 );
-			const filteredItems = result.current.getFilteredItems(
-				result.current.categoriesSelectList,
+			const filteredItems = result.current.getFilteredItemsForSelectTree(
+				result.current.categoriesSelectList.map( ( l ) => ( {
```

Comment:
`l` is undescriptive as a param name. How about something like `listItem`?

Reply:
And, how about moving this mapping function out so it can be shared with the above identical mapping function?

Reply:
Actually, can you just use the exported `mapFromCategoryType()`?